pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -309,9 +309,9 @@ const SourceRow = ({ connectionName, source, sourceType, networks, nodeDevices,
                     {cloudInitSupported ? <FormSelectOption value={CLOUD_IMAGE}
                                       label={_("Cloud base image")} /> : null}
                     <FormSelectOption value={LOCAL_INSTALL_MEDIA_SOURCE}
-                                      label={_("Local install media")} />
+                                      label={_("Local install media (ISO image or distro install tree)")} />
                     <FormSelectOption value={URL_SOURCE}
-                                      label={_("URL")} />
+                                      label={_("URL (ISO image or distro install tree)")} />
```

Comment:
What's a distro install tree?

Is it an archive file with other files inside? Like a .tar / .tgz / or whatever.

Or is it a directory with files?

Or something else?

Will our users know what this means?

Reply:
It's a directory, https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/installation_guide/s1-kickstart2-install-tree

In case they don't know I guess they can google it? It's better than just having that value 'URL' string which does not contain much info.